openjdk11/plan.sh

```diff
@@ -0,0 +1,50 @@
+pkg_origin=core
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_name=openjdk11
+# NOTE: Retrieve download link and shasum from here: https://jdk.java.net/11/
+pkg_version=11.0.2
+pkg_source=https://download.java.net/java/GA/jdk11/9/GPL/openjdk-${pkg_version}_linux-x64_bin.tar.gz
+pkg_shasum=99be79935354f5c0df1ad293620ea36d13f48ec3ea870c838f20c504c9668b57
+pkg_filename=openjdk-${pkg_version}_linux-x64_bin.tar.gz
+pkg_dirname="jdk-${pkg_version}"
+pkg_license=("GPL-2.0")
```

Comment:
Could you update this to `GPL-2.0-only` to align with https://spdx.org/licenses/ .     Looking through the extracted files under `legal/*/LICENSE` it appears that is the `-only` gpl2 string is the correct one. I wasn't able to find any mention of `or later` that would indicate we should use `GPL-2.0-or-later`

Reply:
ah cool, I didn't even know about that!